### ISSUE TITLE
What is the loss of pre-training of the published model?

### ISSUE THREAD

**Issue by Wubeizhongxinghua** [MAGICS-LAB/DNABERT_2#98]
Hello, I am doing pre-training in a similar way based on your code. But it seems the loss decreases very slowly. I am wondering what is the final pre-training loss of the model used for downstream tasks? Thanks!

**Comment by Zhihan1996** [comment]
It was about 4.3. This value highly depends on your vocabulary size.

**Comment by multydoffer** [comment]
> Hello, I am doing pre-training in a similar way based on your code. But it seems the loss decreases very slowly. I am wondering what is the final pre-training loss of the model used for downstream tasks? Thanks!

Hi, Have you pretrained your model successfully?
